recipes/psfgen/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "psfgen" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "ecf5a23805e5fb4841c352203225bd7715e28e9251be2684d1e4c977953b5cbd" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }} -v{{ version }}.tar.gz
+    url: https://github.com/Eigenstate/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 2
```

Until the recipe is first deployed the build number should be 0:
```suggestion
    number: 0
```